gslib/tests/test_context_config.py

```diff
@@ -164,6 +164,10 @@ def setUp(self):
     super(TestContextConfig, self).setUp()
     self.mock_logger = mock.Mock()
 
+  def tearDown(self):
+    super(TestContextConfig, self).tearDown()
+    context_config._singleton_config = None
```

Comment:
If in future we add a test that relies on the context config getting set, then this might create an issue because it will set it to None. To be future proof, maybe we should store the previous value so that this acts like a mock patch.

in setUp:
self._old_context_config = context_config._singleton_config
context_config._singleton_config = None

In tearDown:
context_config._singleton_config = self._old_context_config

Reply:
Done